docs/STYLEGUIDE.md

```diff
@@ -90,7 +90,7 @@ This style guide details the formatting and styling of Daml and client-side code
     deriving (Eq, Ord, Show)
   ```
 
-- Add a `view` field to the interface pureing the view type:
+- Add a `viewtype` definition to the interface:
```

Comment:
I think it has become mandatory now, so we probably don't need this as part of the style guide.

If we want to keep this instruction, we should update the code snippet.